src/pudl/metadata/fields.py

```diff
@@ -3297,6 +3362,10 @@
         "type": "boolean",
         "description": "Whether a plant part record has a duplicate record with different ownership status.",
     },
+    "parent_company_cik": {
+        "type": "string",
+        "description": "CIK of the company's parent company.",
+    },
```

Comment:
What is a CIK? We should spell it out in the description and abbreviate in parentheses.

We have a mix of "company", "parent" and "subsidiary" field names. In any context where there could be any ambiguity as to whether we're referring to a parent or a subsidiary company it should be explicit.

Reply:
I spelled it out in the name since we're using the full `central_index_key` name elsewhere.